src/github/githubRepository.ts

```diff
@@ -893,6 +894,51 @@ export class GitHubRepository implements vscode.Disposable {
 		return ret;
 	}
 
+	async getPullRequestParticipants(pullRequestNumber: number): Promise<IAccount[]> {
+		Logger.debug(`Fetch participants from a Pull Request`, GitHubRepository.ID);
+		const { query, remote, schema } = await this.ensure();
+
+		const ret: IAccount[] = [];
+
+		try {
+			const result: { data: PullRequestParticipantsResponse } = await query<PullRequestParticipantsResponse>({
+				query: schema.GetParticipants,
+				variables: {
+					owner: remote.owner,
+					name: remote.repositoryName,
+					number: pullRequestNumber,
+					first: 15
+				},
+			});
+
+			ret.push(
```

Comment:
Currently I am not sorting the participants as GitHub might have a specific order for them, however, they don't mention an order in the API. Should I just order them Alphabetically?